docs/batch-metrics-api.md

```diff
@@ -0,0 +1,160 @@
+
+Sample Query:
+
+GET localhost:9600/_opendistro/_performanceanalyzer/batch?metrics=CPU_Utilization,IO_TotThroughput&starttime=1594412650000&endtime=1594412665000&samplingperiod=5
+
+Output
+```
+{
+    "1594412650000": {
+        "CPU_Utilization": {
+            "fields": [
+                {
+                    "name": "ShardID",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "IndexName",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "Operation",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "ShardRole",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "sum",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "avg",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "min",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "max",
+                    "type": "DOUBLE"
+                }
+            ],
+            "records": [
+                [
+                    null,
+                    null,
+                    "GC",
+                    null,
+                    0.0039980247804126965,
+                    0.0039980247804126965,
+                    0.0039980247804126965,
+                    0.0039980247804126965
+                ],
+                [
+                    null,
+                    null,
+                    "other",
+                    null,
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ]
+            ]
+        },
+        "IO_TotThroughput": {
+            "fields": [
+                {
+                    "name": "IndexName",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "ShardID",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "ShardRole",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "Operation",
+                    "type": "VARCHAR"
+                },
+                {
+                    "name": "sum",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "avg",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "min",
+                    "type": "DOUBLE"
+                },
+                {
+                    "name": "max",
+                    "type": "DOUBLE"
+                }
+            ],
+            "records": [
```

Comment:
This response should have 2 records right? Your query has a 15s window with a 5s period, and I guess in your particular example you're demonstrating that the final 5s sample may be dropped?

Reply:
Yes, but I think this may be misunderstood by the reader. Also, I don't know how hard the timestamps are to visually differentiate (1594412650000 vs 1594412665000). So, will update this.